finetuner/tuner/dataset/__init__.py

```diff
@@ -138,7 +137,7 @@ def __init__(
                         ' When using ClassDataset all documents need this tag'
                     )
 
-                self._labels.append((i, tag))
+                self._labels.append((i, int(tag)))
```

Comment:
why is this conversion needed? this is quite an assumption right?

Reply:
I am converting because it may be a float (the allowed values are still only `-1` and `1`)